### PR TITLE
refresh build env

### PR DESCRIPTION
**Pull request by stevermeister**
@bampakoa I've fixed playground, please have a look.
but broke test env :(

Plus: fixed focus issue - https://github.com/stevermeister/ngx-wig/issues/47

### CODE REVIEW

**Comment by bampakoa** [comment]
@stevermeister Add (outside of `compilerOptions`)

```
"exclude": [
    "./playground"
  ],
  "include": [
    "./src"
  ]
```

in the `tsconfig.json`

**Comment by bampakoa** [comment]
@stevermeister ignore my previous comment. I am checking your PR right now and I will come back to you. By the way, it [this](http://www.angularplayground.it/) the same playground?

File: tsconfig.json [hunk 1]
```diff
@@ -4,17 +4,9 @@
...
     "skipLibCheck": true,
-    "typeRoots": [
-       "node_modules/@types"	
-     ],	
-     "types": [	
-       "jasmine",	
-       "node"	
-     ]
+    "typeRoots": ["node_modules/@types"],
+    "types": ["jasmine", "node"]
```
Comment (bampakoa) [inline tsconfig.json]:
```
"paths": {
      "ngx-wig": [
        "../dist/ngx-wig"
      ]
    }

otherwise `tsc` command fails
```

**Comment by bampakoa** [comment]
In `./playground/tsconfig.json` add 

```
"baseUrl": ".",
"paths": {
      "ngx-wig": [
        "../dist/ngx-wig"
      ]
}
```

otherwise, the IDE throws an error

In `karma.conf.js` append `"exclude": ["./playground"]` in the **karmaTypescriptConfig** for the tests to pass

**Comment by stevermeister** [comment]
@bampakoa sorry, did not get your point about `tsconfig.json`, could I ask you to make separate PR fix for it?

**Pull request merged**